### DISCUSSION TITLE
X Plane plugin source code

### DISCUSSION THREAD

**Discussion by jesusdparra** [curiosity-workshop/XPlane-Direct-superceded#13] [Q&A]
Hello i have been looking at your project and was wondering if you could share the source code for the plugin for x plane, i would really like to see the way you implemented the serial communication with arduino

**Comment by curiosity-workshop** [comment]
I might be willing to share excerpts.  What are you needing?

**Reply by jesusdparra** [reply]
I'm curious about the serial communication with the arduino module, how did you implemented that?

**Comment by m0boy** [comment]
Hi, I am looking for the same thing, or if you could compile for Mac (Apple Silocon), it would be much appreciated.
